### PR TITLE
Extend seed NFT expiration time

### PR DESCRIPTION
**Pull request by JimAelf**
Ability to allow the owner of the SEED Collection to extend the expiration time for the specified SEED NFT. For issue #3622

### CODE REVIEW

**Comment by JimAelf** [comment]
For issue #3622

**Review by eanzhao — APPROVED** [review]

File: test/AElf.Contracts.MultiToken.Tests/MultiTokenContractTestBase.cs
Comment (chopin-fan) [inline test/AElf.Contracts.MultiToken.Tests/MultiTokenContractTestBase.cs:1]: can remove the changes that didn't take effect

**Review by chopin-fan — APPROVED** [review]
have checked

**Review by David20240718 — APPROVED** [review]
Everything checks out, approved.

**Pull request merged**